packages/fxa-content-server/server/lib/routes/react-app/index.js

```diff
@@ -0,0 +1,84 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const config = require('../../configuration');
+const { getFrontEndRouteDefinitions } = require('./route-definitions');
+
+/** @type {import("./types").RouteFeatureFlagGroup} */
```

Comment:
I spend a solid half-day trying to get Typescript to play nice here. We only have it set up in `app/scripts` for content-server so I tried various things with the tsconfig and otherwise, we're using webpack to pull in `.tscompiled/scripts` but I couldn't get it working and just resorted to JSDoc types. Thankfully we can `import` a TS file for JSDoc so this seems OK for development purposes especially considering we're looking to deprecate all this in the future anyway.